package.json

```diff
@@ -22,10 +22,10 @@
     }
   },
   "scripts": {
-    "dev": "start-storybook -p 6006",
     "build:package": "vite build",
     "build:types": "api-extractor run --local",
     "build": "./scripts/build.bash",
+    "dev": "export NODE_OPTIONS=--openssl-legacy-provider && start-storybook -p 6006",
```

Comment:
what is this for?

Reply:
ref: https://stackoverflow.com/a/69699772/4962244

Reply:
seen that, not a huge fan, I'm trying to find another fix,

Reply:
ok, I will remove the changes